sdk/src/main/kotlin/io/rover/sdk/Rover.kt

```diff
@@ -134,6 +138,16 @@ open class Rover(
 
     open val eventEmitter: EventEmitter = EventEmitter(LocalBroadcastManager.getInstance(application)),
 
+    open val installationIdentification: InstallationIdentification = InstallationIdentification(application),
+
+    open val eventAnalyticsService: EventAnalyticsService = EventAnalyticsService(
+        URL(analyticsEndpoint),
```

Comment:
actually thinking we should just the analytics endpoint inside EventAnalyticsService itself as a constant.